src/js/claims-status/utils/appeals-v2-helpers.jsx

```diff
@@ -530,121 +530,337 @@ export function getEventContent(event) {
+                by the Veterans Benefits Administration.
+              </p>
+            ),
+            durationText: '',
+            cardDescription: '',
+          }
+        ]
+      };
+    case STATUS_TYPES.pendingCertificationSsoc:
+      return  {
+        header: 'What happens next depends on whether you send in new evidence.',
+        events: [
+          {
+            title: 'Your appeal will be sent to the Board',
+            description: (
+              <p>
+                <strong>If you don’t send in new evidence after the Statement of the Case on [DATE]</strong>,
+                the Decision Review Officer will finish their review and transfer your case to the
+                Board of Veterans’ Appeals.
+              </p>
+            ),
+            durationText: '',
+            cardDescription: '',
+          }, {
+            title: 'The Veterans Benefits Administration will send you a new Statement of the Case',
+            description: (
+              <p>
+                <strong>If you send new evidence after the Statement of the Case on [DATE]</strong>, the
+                Decision Review Officer will need to write a new Statement of the Case before
+                transferring your case to the Board of Veterans’ Appeals. Once your appeal is
+                transferred, new evidence can be sent directly to the Board and will not be reviewed
+                by the Veterans Benefits Administration.
+              </p>
+            ),
+            durationText: '',
+            cardDescription: '',
+          }
+        ]
+      };
+    case STATUS_TYPES.remandSsoc:
+      return {
+        header: 'What happens next depends on whether you send in new evidence.',
+        events: [
+          {
+            title: 'Your appeal will be returned to the Board',
+            description: (
+              <p>
+                <strong>If you don’t send in new evidence after the Statement of the Case on [DATE]</strong>,
+                the Veterans Benefits Administration will finish their work on the remand and return
+                your case to the Board of Veterans’ Appeals.
+              </p>
+            ),
+            durationText: '',
+            cardDescription: '',
+          }, {
+            title: 'The Veterans Benefits Administration will send you a new Statement of the Case',
+            description: (
+              <p>
+                <strong>If you send in new evidence after the Statement of the Case on [DATE]</strong>, the
+                Veterans Benefits Administration will need to write a new Statement of the Case
+                before returning your case to the Board of Veterans’ Appeals.
+              </p>
+            ),
+            durationText: '',
+            cardDescription: '',
+          }
+        ]
+      };
+    case STATUS_TYPES.pendingHearingScheduling:
+      return {
+        header: '', // intentionally empty
```

Comment:
From what I could tell, this didn't actually render an empty tag - I think React is being smart under the hood. I debated between optionally returning a `header` property vs always returning a `header` property and leaving it empty when required. I ultimately chose this path because it makes the fact that there is no header text very explicit, versus perhaps wondering (6 months later) whether we accidentally left the header off a content snippet. I'm happy to revisit if you think this is the wrong approach - but I'll at least double check that this doesn't render an empty `<p/>` element.

Reply:
Awesome. Sounds good!